src/main/scala/uk/gov/hmrc/scalatestaccessibilitylinter/config.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.scalatestaccessibilitylinter
+
+import com.typesafe.config.ConfigFactory
+import uk.gov.hmrc.scalatestaccessibilitylinter.domain._
+import uk.gov.hmrc.scalatestaccessibilitylinter.gateways._
+import uk.gov.hmrc.scalatestaccessibilitylinter.linters._
+
+object config {
+  private lazy val config              = ConfigFactory.parseResources(getClass, "accessibility-linter.conf")
+  lazy val knownIssues: KnownIssues    = KnownIssues.fromConfigList(config.getConfigList("known-issues"))
+  lazy val defaultVnuLinter: VnuLinter = embeddedVnuLinter
+  lazy val defaultAxeLinter: AxeLinter = localNpmWrapperAxeLinter
+  lazy val embeddedVnuLinter           = new VnuLinter(Vnu.embedded, knownIssues)
```

Comment:
Potentially provide a VnuLinter object within VnuLinter.scala that does the necessary configuration?